src/fn/csv-to-docs.js

```diff
@@ -34,7 +34,7 @@ module.exports = (projectDir)=> {
     exclusions: [],
   };
   const addToModel = (csvFile, docs) => {
-    csvFile = csvFile.match(/^(?:.*[\/\\])?csv[\/\\](.*)\.csv$/)[1];
+    csvFile = csvFile.match(/^(?:.*[\/\\])?csv[\/\\](.*)\.csv$/)[1]; // eslint-disable-line no-useless-escape
```

Comment:
Or fix the useless escape? I'm guessing it's for the `\/` inside the square brackets

Reply:
Do you mind if I open issues and do separate PRs for changes which might impact code function. I'd like to do those a bit more methodically. This code has no unit tests .. so just want to be cautious and not break things.

Reply:
Yeah no worries